src/azul/chalice.py

```diff
@@ -13,9 +14,12 @@ def route(self, path, **kwargs):
         Same as method in supper class but stashes URL path a view function is bound to as an attribute of the
         function itself.
         """
+        description = kwargs.pop('spec', None)
         decorator = super().route(path, **kwargs)
 
         def _decorator(view_func):
+            if description:
+                spec(description)(view_func)
```

Comment:
```suggestion
                view_func = spec(description)(view_func)
```

This is the decorator protocol. If you didn't assign, you'd be relying on an implementation detail of `spec`.

Reply:
That's what I had originally, but I realized it was redundant. I changed it back however.